.github/workflows/release_update_docs_px_dev.yaml

```diff
@@ -0,0 +1,102 @@
+---
+name: update-docs-px-dev-on-release
+on:
+  push:
+    tags:
+    - release/vizier/**
+permissions:
+  contents: read
+jobs:
+  get-dev-image:
+    uses: ./.github/workflows/get_image.yaml
+    with:
+      image-base-name: "dev_image_with_extras"
+  generate-docs:
+    needs: get-dev-image
+    runs-on: [self-hosted, nokvm]
```

Comment:
can you take a look at the new `build-and-test` and follow that for removing `self-hosted`? We're transitioning away from `self-hosted` so it would be nice to not add a new one.

Reply:
Since this is not run on PRs, you don't need any of the authorize, or env-protect-setup stuff, just the `use_remote_exec` and `BB_API_KEY` in the bazelrc action, as well as running on `ubuntu-latest-{8,16}-cores` instead of self-hosted

Reply:
Sounds good. I've updated this in the latest version and went with 8 cores since this job doesn't need to run as fast.